app/src/main/cpp/DeviceUtils.cpp

```diff
@@ -16,10 +16,16 @@ std::unordered_map<std::string, device::DeviceType> DeviceUtils::deviceNamesMap;
 
 vrb::Matrix
 DeviceUtils::CalculateReorientationMatrix(const vrb::Matrix& aHeadTransform, const vrb::Vector& aHeightPosition) {
-  const float kPitchUpThreshold = 0.2f;
-  const float kPitchDownThreshold = 0.5f;
-  const float kRollThreshold = 0.35f;
+  return CalculateReorientationMatrixWithThreshold(aHeadTransform, aHeightPosition, 0.2f, 0.5f, 0.35f);
+}
 
+vrb::Matrix
+DeviceUtils::CalculateReorientationMatrixTracking(const vrb::Matrix& aHeadTransform, const vrb::Vector& aHeightPosition) {
```

Comment:
Nit: CalculateReorientationMatrixOnHeadLock

Reply:
Fixed the merge conflict, rebased, and corrected this one.